doc/source/user_guide/ug_case_settings.rst

```diff
@@ -362,12 +364,15 @@ Table of namelist options
    "", "``monthly``", "monthly forcing data", ""
    "", "``ncar``", "NCAR bulk forcing data", ""
    "", "``oned``", "column forcing data", ""
+   "", "``hycom_atm``", "HYCOM atm forcing in NetCDF format", ""
    "``ocn_data_type``", "``default``", "constant values defined in the code", ""
    "", "``clim``", "climatological data", ""
    "", "``ncar``", "POP ocean forcing data", ""
+   "", "``hycom``", "HYCOM ocean forcing data.", "Constant initial forcing"
    "``bgc_data_type``", "``default``", "constant values defined in the code", ""
    "", "``clim``", "climatological data", ""
    "", "``near``", "POP ocean forcing data", ""
```

Comment:
Can you also go ahead and change this typo with this PR. It should be "ncar" not "near" as the option. Thanks!